src/main/kotlin/uk/gov/gdx/datashare/service/LegacyAdaptorInbound.kt

```diff
@@ -38,38 +33,6 @@ class LegacyAdaptorInbound(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @Scheduled(fixedRate = 1, timeUnit = TimeUnit.MINUTES)
```

Comment:
Do we want to remove this? We can leave this in as an example of how to connect to an FTP server if we comment out the schedule and add a comment explaining what it is

Reply:
I think remove